app/services/container_topology_service.rb

```diff
@@ -114,15 +114,12 @@ def retrieve_providers
   end
 
   def build_kinds
-    kinds =  { :Replicator => true,
-               :Pod        => true,
-               :Container  => true,
-               :Node       => true,
-               :Service    => true,
-               :Host       => true,
-               :VM         => true,
-               :Route      => true
-             }
+    base_kinds = [:Replicator, :Pod, :Container, :Node, :Service, :Host, :VM, :Route]
+    kinds = {}
+    base_kinds.each do |kind|
+      kinds[kind] = true
+    end
+
     if @providers.any? { |instance| instance.kind_of?(ManageIQ::Providers::Kubernetes::ContainerManager) }
       kinds.merge!(:Kubernetes => true)
```

Comment:
`base_kinds << :Kubernetes` (and move the hashification to the end)? (Probably rename `base_kinds` to `kinds` and just make the last line `kinds.each_with_object({}) { |kind, h| h[kind] = true }`, so no second variable... but that's detail.)

Seems slightly cleaner to keep the same perspective for all of them.

I do like this change... makes it clearer that the hash is secondary / the values don't vary.